Add material-select for the defined site-language

NiklasFLuis opened the pull request:
### description
Add a material-select to the site-information to set the `lang`-attribute of current site. Available choices are 'en' and 'de'. Only the choice from the template is used.

### images
<img width="315" alt="lang-select" src="https://user-images.githubusercontent.com/51320830/63341762-3bd01680-c34a-11e9-8397-32ba1b3079d4.png">
<img width="203" alt="lang-choices" src="https://user-images.githubusercontent.com/51320830/63341958-a5e8bb80-c34a-11e9-9082-e41e0ed38af5.png">

<img width="123" alt="lang-tag" src="https://user-images.githubusercontent.com/51320830/63341850-6d48e200-c34a-11e9-8ce8-c10690b46628.png">

NiklasFLuis committed:
METDEV-1389: add new action to nav in admin-edit-page

NiklasFLuis committed:
METDEV-1389: update action in admin-edit-page

NiklasFLuis committed:
METDEV-1389: add download to action and update nav

NiklasFLuis committed:
METDEV-1389: update download propertie in action

NiklasFLuis committed:
METDEV-1389: update download property in action

NiklasFLuis committed:
METDEV-1389: update download property in action

NiklasFLuis committed:
METDEV-1389: update download property in action

change to pageView/path/jcr:content

NiklasFLuis committed:
METDEV-1389: update download property in action

change to pageView/path

NiklasFLuis committed:
METDEV-1389: update download property in action

use perAdminApp to get path

NiklasFLuis committed:
METDEV-1389: add dropdown-component to admin-base

NiklasFLuis committed:
METDEV-1389: add dropdown-component to admin-base

NiklasFLuis committed:
METDEV-1389: update dropdown-component in admin-base

NiklasFLuis committed:
METDEV-1389: update download action

NiklasFLuis committed:
METDEV-1389: update download action

add title

NiklasFLuis committed:
METDEV-1389: update download action

add title
add extra icons

NiklasFLuis committed:
METDEV-1389: update download action

add extra icons
remove dropdown

NiklasFLuis committed:
METDEV-1389: update download action

change icon

NiklasFLuis committed:
METDEV-1389: update download action

change icon

NiklasFLuis committed:
METDEV-1389: update download action

change behavior. will now open .xml in new page

NiklasFLuis committed:
METDEV-1389: update download action

add download-text action
add dropdown in edit-page

NiklasFLuis committed:
METDEV-1389: add DropdownModel.java

NiklasFLuis committed:
METDEV-1389: update dropdown/template.vue

add data-per-path prop

NiklasFLuis committed:
METDEV-1389: update edit/.context.xml

remove jcr:content from dropdown

NiklasFLuis committed:
METDEV-1389: remove dropdown

NiklasFLuis committed:
METDEV-1389: add export-action to main Navigation

remove export-action from sub-nav

NiklasFLuis committed:
METDEV-1389: move export-action to sub-nav template.vue

NiklasFLuis committed:
METDEV-1389: add seperator between export-action and other actions

NiklasFLuis committed:
METDEV-1389: add data-uri to download xml-file

NiklasFLuis committed:
METDEV-1389: revert data-uri

NiklasFLuis committed:
METDEV-1389: add tooltipTitle

NiklasFLuis committed:
METDEV-1389: fix typo in admin-components-subnav: separator

NiklasFLuis committed:
METDEV-1389: change downloaded file

NiklasFLuis committed:
METDEV-1389: add download-attribute to action

NiklasFLuis committed:
METDEV-1389: change download-attribute in action

NiklasFLuis committed:
METDEV-1389: remove unused download-action

NiklasFLuis committed:
METDEV-1389: remove unused dropdown-component

NiklasFLuis committed:
METDEV-1389: remove unused dropdown-model

NiklasFLuis committed:
Merge branch 'metaproc-develop' into feature/METDEV-1389

NiklasFLuis committed:
METDEV-1389: add download-prop to action

Commit:
METDEV-1313: Create SiteMapServlet; extend per:PageContent with sitemap specific properties

Commit:
METDEV-1313: Create SiteMapServlet; Refactor ResourceEventHandler

DimitriPlotnikov committed:
Merge pull request #13 from meta-proc/feature/METDEV-1312

feature/METDEV-1312

NiklasFLuis committed:
Merge branch 'metaproc-develop' into feature/METDEV-1374

Commit:
METDEV-1385 add headline field

Commit:
METDEV-1313: Finish SitemapServlet; Refactor PageEventHandlerService

Commit:
METDEV-1313: Removed SitemapIndex servlet

Commit:
METDEV-1313: Fix lastMod, default to created property

DimitriPlotnikov committed:
Merge pull request #11 from meta-proc/feature/METDEV-1389

METDEV-1389

NiklasFLuis committed:
Merge branch 'metaproc-develop' into feature/METDEV-1374

felixulbrich committed:
METDEV-1267 Add necessary changes from old branch

felixulbrich committed:
METDEV-1267 Add changes to action component from old branch

DimitriPlotnikov committed:
Merge pull request #16 from meta-proc/feature/METDEV-1267_v2

METDEV-1267 Edit-page: Encapsulate screen width selection items into a dropdown menu

Commit:
Merge remote-tracking branch 'upstream/develop' into metaproc-develop

# Conflicts:
#	admin-base/ui.apps/src/main/content/jcr_root/content/admin/pages/.content.xml
#	admin-base/ui.apps/src/main/js/stateActions/editPreview.js

Commit:
METDEV-1408 clear translations of current language before switching

Commit:
METDEV-1408 clear translations of current language before switching

Commit:
Revert "Update .travis.yml"

This reverts commit 77e346c04442ae23f5097aa5b8f775c0977230d0.

Commit:
revert travis.yml

DimitriPlotnikov committed:
Merge pull request #15 from meta-proc/feature/METDEV-1313

feature/METDEV-1313

NiklasFLuis committed:
METDEV-1374: add metaproc-components to .gitignore

DimitriPlotnikov committed:
Merge pull request #18 from meta-proc/bug/METDEV-1408_metaproc-develop

bug/METDEV-1408

Commit:
Merge branch 'feature/METDEV-1385' into metaproc-develop

* feature/METDEV-1385:
  revert travis.yml
  METDEV-1385 add headline field
  Update .travis.yml
  Revert "feature/METDEV-1312"

# Conflicts:
#	admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/nav/template.vue
#	admin-base/ui.apps/src/main/content/jcr_root/content/admin/pages/edit/.content.xml
#	admin-base/ui.apps/src/main/js/stateActions/editPreview.js
#	pagerenderer/vue/ui.apps/src/main/content/jcr_root/apps/pagerender/vue/structure/page/explorer_dialog.json
#	platform/base/core/src/main/java/com/peregrine/listener/PageEventHandlerService.java
#	platform/base/core/src/main/java/com/peregrine/seo/impl/UrlExternalizerImpl.java
#	platform/base/core/src/main/resources/SLING-INF/nodetypes/pagecontent.cnd
#	platform/commons/src/main/java/com/peregrine/commons/util/PerConstants.java

Commit:
Merge branch 'develop' into metaproc-develop

Commit:
apply code format

Commit:
Revert "Merge pull request #18 from meta-proc/bug/METDEV-1408_metaproc-develop"

This reverts commit 5067b6aeac9c6667a046db976357dec89285f789, reversing
changes made to b98f54a2f64f749bb7d8bc813532463c69f18d17.

Commit:
Merge branch 'metaproc-develop' of https://github.com/meta-proc/peregrine-cms into metaproc-develop

* 'metaproc-develop' of https://github.com/meta-proc/peregrine-cms:
  revert travis.yml
  METDEV-1385 add headline field
  Update .travis.yml
  Revert "feature/METDEV-1312"

Commit:
Revert "Merge branch 'metaproc-develop' of https://github.com/meta-proc/peregrine-cms into metaproc-develop"

This reverts commit 86412047197a52414511e7c8f3d000ff13f311e2, reversing
changes made to c2a100530cf98d25ac4d04f14effcaa8b84741ab.

Commit:
METDEV-1385 add headline field

Commit:
METDEV-1385 register headline field

flx-sta committed:
Merge pull request #20 from meta-proc/merged/METDEV-1385_metaproc-develop

METDEV-1385 PeregrineCMS: Bedienelement: Headline implementieren

DimitriPlotnikov committed:
Update .travis.yml

DimitriPlotnikov committed:
Merge pull request #21 from meta-proc/DimitriPlotnikov-patch-1

Update .travis.yml

Commit:
Merge branch 'develop' into metaproc-develop

Commit:
METDEV-1331 add new admin constant "PathBrowser"
- abb subobject "Type" with values PAGE, ASSET, IMAGE

Commit:
METDEV-1331 add new browserType "image" for pathbrowser
- default of "withLink" for image browser is "false"
- for other all other browser types it stays "true"

Commit:
METDEV-1331 replace single quotation marks with double (for imports!)

Commit:
METDEV-1331 remove console.log()

Commit:
METDEV-1331 reactivate search for image browser
- deactivate choosing non-image files

Commit:
METDEV-1475 make textedit btns and btnsDef customizable

Commit:
METDEV-1475 make textedit btns and btnsDef customizable

Commit:
METDEV-1475 fix "can read btns of undefined" error

Commit:
METDEV-1475 fix "can read btns of undefined" error

NiklasFLuis committed:
METDEV-1374: add placeholder for customcomponents in mdbvueLoaderFree.js

felixulbrich committed:
METDEV-1473 Try adjusting z-index to make dropdown visible in preview mode

DimitriPlotnikov committed:
Merge pull request #23 from meta-proc/METDEV-1475

METDEV-1475 Textedit Feld muss konfigurierbar sein

DimitriPlotnikov committed:
Merge pull request #24 from meta-proc/feature/METDEV-1374

METDEV-1374

felixulbrich committed:
METDEV-1473
Remove unnecessary change from last commit.
Change z-index for workspace from 1 to auto

felixulbrich committed:
Merge branch 'metaproc-develop' into bug/METDEV-1427

Commit:
Merge branch 'metaproc-develop' into METDEV-1331

* metaproc-develop:
  METDEV-1374: add placeholder for customcomponents in mdbvueLoaderFree.js
  METDEV-1475 fix "can read btns of undefined" error
  METDEV-1475 make textedit btns and btnsDef customizable
  METDEV-1374: add metaproc-components to .gitignore

Commit:
METDEV-1331 add some debug output to locate bug

Commit:
METDEV-1331 add some debug output to locate bug (fix)

Commit:
METDEV-1331 debugging

Commit:
METDEV-1331 all images are now selectable in browserType image
- even images that have the wrong mime-type configured
- improved concept to check if an item is selectable

DimitriPlotnikov committed:
Merge pull request #27 from meta-proc/bug/METDEV-1473

METDEV-1473 - Make screen width dropdown visible in preview mode

Commit:
METDEV-1331 add svg to the image extension list

Commit:
Merge branch 'metaproc-develop' into METDEV-1331

* metaproc-develop:
  METDEV-1473 Remove unnecessary change from last commit. Change z-index for workspace from 1 to auto
  METDEV-1473 Try adjusting z-index to make dropdown visible in preview mode

flx-sta committed:
Merge pull request #25 from meta-proc/METDEV-1331

METDEV-1331

Commit:
Merge branch 'develop' into merging/metaproc-develop

* develop:
  #134: properly initializing editor state when editing a page
  Feature/pagerissue (#127)
  #125: assets now pulls its model.json from an explorer_model.json (#126)
  #109: make sure per:PageContent does not get altered when saving a page property (#122)
  issues/103: implement movePage using pathbrowser (#132)
  Issues/54 (#61)
  sort components in group
  applied resetcss to editor
  fix for hover when dropping components
  fix for escaping </script> tags in content
  change border color when hoevring over a component from a template
  #118: move to include the data.json into the html to limit requests (#119)
  remove output of rollup as it was logging to error
  updated bootstrap and popper.js
  remove pace (blue progress bar at the top of rendered vue pages)

# Conflicts:
#	admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/assetview/template.vue
#	admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/pathfield/template.vue
#	admin-base/ui.apps/src/main/content/jcr_root/apps/field/texteditor/template.vue
#	pagerenderer/vue/ui.apps/package.json
#	platform/commons/src/main/java/com/peregrine/commons/util/PerConstants.java

Commit:
Merge branch 'develop' into merging/metaproc-develop

* develop:
  #115: allow for adding components between containers
  fix for incorrect .content.xml file
  Revert "Issues/54 (#61)" (#138)

# Conflicts:
#	platform/commons/src/main/java/com/peregrine/commons/util/PerConstants.java

Commit:
METDEV-1452 add trumbowyg colors

Commit:
METDEV-1452 add trumbowyg colors

Commit:
METDEV-1452 add "plugins" to config

Commit:
METDEV-1452 add "plugins" to config

NiklasFLuis committed:
METDEV-1509: fix bug with export-button

move target and download to own functions and only display components on certain pages (v-if)

Commit:
METDEV-1314: Refactor PageEventHandlerService

Commit:
hotfix: viewport dropdown on template-editor

DimitriPlotnikov committed:
Merge pull request #30 from meta-proc/feature/METDEV-1509

METDEV-1509

Commit:
METDEV-1314: Create RobotsServlet to generate robots.txt

NiklasFLuis committed:
METDEV-1282: add new field-controls

will generate a number of buttons from array list and hook up an event to $perAdminApp

NiklasFLuis committed:
Merge branch 'metaproc-develop' into feature/METDEV-1282

Commit:
Merge branch 'develop' into feature/METDEV-1475

* develop:
  #115: allow for adding components between containers
  fix for incorrect .content.xml file
  Revert "Issues/54 (#61)" (#138)
  #134: properly initializing editor state when editing a page
  Feature/pagerissue (#127)
  #125: assets now pulls its model.json from an explorer_model.json (#126)
  #109: make sure per:PageContent does not get altered when saving a page property (#122)
  issues/103: implement movePage using pathbrowser (#132)
  Issues/54 (#61)
  sort components in group
  applied resetcss to editor
  fix for hover when dropping components
  fix for escaping </script> tags in content
  change border color when hoevring over a component from a template
  #118: move to include the data.json into the html to limit requests (#119)
  remove output of rollup as it was logging to error
  updated bootstrap and popper.js

# Conflicts:
#	admin-base/ui.apps/src/main/content/jcr_root/apps/field/texteditor/template.vue

Commit:
Merge branch 'develop' into metaproc-develop

* develop:
  #151: fix for paste action in edtior (was not checking for right class)
  #73: removed delete when path of component is /jcr:content (#121)
  #100: fix for pasting components (#143)
  add list styles to trumbo reset css (#145)

NiklasFLuis committed:
Merge branch 'metaproc-develop' into feature/METDEV-1282

Commit:
Merge remote-tracking branch 'origin/metaproc-develop' into feature/METDEV-1314

Commit:
METDEV-1314: Small fix and merged remote metaproc-develop into current branch

reusr1 committed:
Merge branch 'develop' into feature/METDEV-1311

Commit:
Merge branch 'develop' into metaproc-develop

* develop:
  textedit config customizable (#117)
  Fix: double language switch not recognized (#112)
  fix for incorrect path separator in suffix
  action now does not link to [object] anymore
  read tags properly on page

# Conflicts:
#	admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/action/template.vue
#	admin-base/ui.apps/src/main/content/jcr_root/apps/field/texteditor/template.vue

Commit:
Merge branch 'METDEV-1452' into metaproc-develop

* METDEV-1452:
  METDEV-1452 add "plugins" to config
  METDEV-1452 add trumbowyg colors

Commit:
add cleanpaste trumbowyg plugin to node-modules.txt

Commit:
METDEV-1514 outsourced ignoreContainers state into constant
- makes the code cleaner

Commit:
METDEV-1514 replace double quotes with single quotes (in import)
- remove semicolons to match the existing code

Commit:
METDEV-1514 add getter for ignoreContainers to fix error:
- "Cannot read property 'ignoreContainers' of undefined"

Commit:
METDEV-1514 add additional check for ignoreContainers to fix error:
- "Cannot read property 'ignoreContainers' of undefined"

DimitriPlotnikov committed:
Merge pull request #33 from meta-proc/METDEV-1514

METDEV-1514 TypeError: Cannot read property 'ignoreContainers' of undefined

DimitriPlotnikov committed:
Merge pull request #32 from meta-proc/feature/METDEV-1314

METDEV-1314: Automatic creation of robots.txt

DimitriPlotnikov committed:
Merge pull request #31 from meta-proc/feature/METDEV-1282

METDEV-1282: Improve carousel component

reusr1 committed:
Merge branch 'develop' into feature/METDEV-1311

Commit:
METDEV-1508 changed placeholder width from 100% to auto

DimitriPlotnikov committed:
Merge pull request #34 from meta-proc/METDEV-1508

METDEV-1508

NiklasFLuis committed:
METDEV-1530: add EventBus to possible Vue-initialization

NiklasFLuis committed:
METDEV-1530: remove EventBuss from ssr.js. Did not work

NiklasFLuis committed:
METDEV-1530: add $eventBus to peregrineApp and $adminEventBus to perAdminApp

NiklasFLuis committed:
METDEV-1530:  change event to send to $adminEventBus

NiklasFLuis committed:
METDEV-1530: move constant EventBus to its own file and import in peregrineApp.js

NiklasFLuis committed:
METDEV-1530: move constant AdminEventBus to its own file and import it in perAdminApp.js

Aaron-Seggelke committed:
METDEV-1571 added horizontal line option

DimitriPlotnikov committed:
Merge pull request #35 from meta-proc/METDEV-1530

Metdev 1530

Commit:
Merge branch 'develop' into metaproc-develop

* develop:
  autodetect java
  METDEV-1508 changed placeholder width from 100% to auto
  pegged percli version to 0.1.26-alpha due to an issue building the docker image with the latest version
  #161 - Updated the Docker image to install the VIPS Linux packages (#162)
  fix issue with incorrect xml
  added index page and updated copyright year in themeclean
  Made sure that Image Transformations are processed even if they are disabled or VIPS not installed
  Added a path to the Image Transformation to make it possible to select them based on the asset path
  removed some of those pesky console.log statements
  german translation (& i18n non case sensitive) (#95)
  added version display to slingpackger for the logs
  updated to use @peregrinecms/slingpackager instead of a curl
  fix themeclean images
  action now supports a visibility expression (#154)
  Added Image Scaling to the Image Transformation and fixed an issue with duplicate Image Transformations when updating the configuration
  Issue/124 (#147)
  added is PublicFacing check to make sure service worker only is enabled on public sites

# Conflicts:
#	admin-base/ui.apps/src/main/content/jcr_root/content/admin/docs/.content.xml
#	admin-base/ui.apps/src/main/content/jcr_root/content/admin/pages/.content.xml
#	admin-base/ui.apps/src/main/content/jcr_root/content/admin/pages/edit/.content.xml
#	admin-base/ui.apps/src/main/content/jcr_root/content/admin/templates/edit/.content.xml
#	admin-base/ui.apps/src/main/content/jcr_root/content/admin/tools/.content.xml
#	admin-base/ui.apps/src/main/content/jcr_root/i18n/admin/de/components/.content.xml
#	admin-base/ui.apps/src/main/content/jcr_root/i18n/admin/de/info/.content.xml
#	admin-base/ui.apps/src/main/content/jcr_root/i18n/admin/de/select parent/.content.xml
#	pagerenderer/vue/ui.apps/src/main/content/jcr_root/apps/pagerender/vue/components/placeholder/template.vue

Commit:
METDEV-1570 fix ignoreContainer error on dragStart/-Stop

Commit:
METDEV-1570 fix ignoreContainer error on btn click

Commit:
METDEV-1570 fix "view" is not defined

Commit:
METDEV-1570 remove ignoreContainers button beeing clickable in preview

NiklasFLuis committed:
METDEV-1613: fix bug with eventBus on public-site

Aaron-Seggelke committed:
METDEV-1571 added horizontal line to toolingpage html

Aaron-Seggelke committed:
Merge branch 'develop' into METDEV-1571

NiklasFLuis committed:
Bug-fix rollup path in build-command

Fix build-command using wrong path to rollup

flx-sta committed:
Merge pull request #39 from meta-proc/bugfix/rolluppath

Bug-fix rollup path in build-command

DimitriPlotnikov committed:
Merge pull request #36 from meta-proc/METDEV-1570

METDEV-1570 IgnoreContainers button highlight kann im preview invertiert werden

DimitriPlotnikov committed:
Merge pull request #37 from meta-proc/METDEV-1613

METDEV-1613: fix bug with eventBus on public-site

Aaron-Seggelke committed:
Merge branch 'metaproc-develop' into METDEV-1571

Commit:
Merge branch 'merged/METDEV-1311' of https://github.com/meta-proc/peregrine-cms into metaproc-develop

# Conflicts:
#	admin-base/ui.apps/src/main/content/jcr_root/content/admin/pages/.content.xml

Commit:
Merge branch 'merged/METDEV-1408' of https://github.com/meta-proc/peregrine-cms into metaproc-develop

Commit:
Merge branch 'merged/METDEV-1311' of https://github.com/meta-proc/peregrine-cms into metaproc-develop

# Conflicts:
#	admin-base/ui.apps/src/main/content/jcr_root/content/admin/pages/.content.xml

DimitriPlotnikov committed:
Merge pull request #38 from meta-proc/METDEV-1571

METDEV-1571

NiklasFLuis committed:
METDEV-1659: Fix problem with rollup

Fix problem with bath of Rollup being changed in module

NiklasFLuis committed:
METDEV-1659: Add five choices to multiselect

Add five choicec to multiselect to be able to differenitate which
deklaration is currently affecting the template. Choices are
en-EN as default by Peregrine-CMS, de-DE as default set in
'pagecontent.cnd', portuguese set in site-.content.xml and spanish set
in template-.content.xml

NiklasFLuis committed:
METDEV-1659: Add siteLanguage to page.html

Add siteLanguage to page.html but it will still get set individually

NiklasFLuis committed:
METDEV-1659: Make material-select template only

Make material-select for language only available in templates.

NiklasFLuis committed:
METDEV-1659: Add siteLanguage to helper

Add private variable 'siteLanguage' to Helper.java and access it in
page.html.

Add initialization for 'siteLanguage' in init()-function by using the
value set in bindings->properties->siteLanugage its default value is set
to 'en-EN', if no value is provided

NiklasFLuis committed:
METDEV-1659: Add function to get template value

Add a function to PageModel to be able to get the template-siteLanguage
value from within the Helper.java. This way 'siteLanguage' in Helper
will always be set to the value which is either set in template
.content.xml or the value which was set in the editor-template-info.

NiklasFLuis committed:
METDEV-1659: Change explorer dialog

Remove not working property visible. Was originally meant to exclude the
material-select from sites, but it throws errors in browser console.

Remove additional languages that were used to differentiate which
language declaration is providing the current site value.

NiklasFLuis committed:
METDEV-1659: Reset language to simple version

Reset used language to its simple declaration. 'en' instead of 'en-US'
and 'de' instead of 'de-DE'.

NiklasFLuis committed:
METDEV-1659: Reset rollup path in package json

NiklasFLuis committed:
Merge branch 'develop' into METDEV-1659

NiklasFLuis committed:
METDEV-1659: Fix hint for material-select

Commit:
Merge branch 'develop' into metaproc-develop

* develop: (127 commits)
  added ability to direcly edit files with composum in the page tree
  Feature/sitedeletion (#179)
  added composum under tools
  Added function to create a package of all default site content when new sites are created. (#185)
  added path to output of tags and metaproperties
  another packagelock
  added package lock
  Updated Docker build script to exit the build if the Maven fails.
  Revert "Merge branch 'develop-sling11-merge' into develop"
  Added logic to site copy/creation process to update strings in files after copy. (#178)
  action now supports a visibility expression (#174)
  issues/175 - Updated package install script to handle installation of any new Peregrine packages.
  issues/175 - Initial work to discover and deploy all Peregrine packages during the Docker build.
  Rebuilt Peregrine Sling instance.
  improve drop target styles (#167)
  Updated Docker build to remove generated artifacts from files directory.
  fix for i18n issue causing all english terms to be lower cased
  switch from select to material-select for the metadata type
  combining meta name and properties into one field in the dialog
  updating the title for meta properties content
  ...

# Conflicts:
#	pagerenderer/vue/ui.apps/src/main/content/jcr_root/apps/pagerender/vue/components/placeholder/template.vue
#	pagerenderer/vue/ui.apps/src/main/content/jcr_root/apps/pagerender/vue/structure/page/explorer_dialog.json

Commit:
fix rollup directory to fix run script

reusr1 commented on platform/base/core/src/main/resources/SLING-INF/nodetypes/pagecontent.cnd:
```diff
@@ -26,4 +26,4 @@
 [per:PageContent] > nt:unstructured, sling:Resource, sling:VanityPath, per:Replication
   orderable
-
+  - siteLanguage (string) = "de" autocreated
```
it may be better to set the default to en

NiklasFLuis replied:
will change it, forgot to set it to english

reusr1 commented on pagerenderer/vue/ui.apps/src/main/content/jcr_root/apps/pagerender/vue/structure/page/explorer_dialog.json:
```diff
@@ -15,6 +15,20 @@
   },
+  {
+    "type": "material-select",
+    "label": "lang=",
```
should we really use lang= as the label for this field?

NiklasFLuis replied:
I was thinking about a label like "Site Language"

NiklasFLuis committed:
METDEV-1659: Change label and default language

Change material-select label to be a usefull description instead of just
'lang='

Change default language for created sites to 'en' instead of 'de'

Commit:
Merge branch 'metaproc-develop' into METDEV-1659

* metaproc-develop: (105 commits)
  fix rollup directory to fix run script
  Bug-fix rollup path in build-command
  METDEV-1571 added horizontal line to toolingpage html
  METDEV-1613: fix bug with eventBus on public-site
  METDEV-1570 remove ignoreContainers button beeing clickable in preview
  METDEV-1570 fix "view" is not defined
  METDEV-1570 fix ignoreContainer error on btn click
  METDEV-1570 fix ignoreContainer error on dragStart/-Stop
  METDEV-1571 added horizontal line option
  METDEV-1530: move constant AdminEventBus to its own file and import it in perAdminApp.js
  METDEV-1530: move constant EventBus to its own file and import in peregrineApp.js
  METDEV-1530:  change event to send to $adminEventBus
  METDEV-1530: add $eventBus to peregrineApp and $adminEventBus to perAdminApp
  METDEV-1530: remove EventBuss from ssr.js. Did not work
  METDEV-1530: add EventBus to possible Vue-initialization
  METDEV-1508 changed placeholder width from 100% to auto
  METDEV-1514 add additional check for ignoreContainers to fix error: - "Cannot read property 'ignoreContainers' of undefined"
  METDEV-1514 add getter for ignoreContainers to fix error: - "Cannot read property 'ignoreContainers' of undefined"
  METDEV-1514 replace double quotes with single quotes (in import) - remove semicolons to match the existing code
  METDEV-1514 outsourced ignoreContainers state into constant - makes the code cleaner
  ...

# Conflicts:
#	platform/base/core/src/main/resources/SLING-INF/nodetypes/pagecontent.cnd

flx-sta commented:
@NiklasFLuis please crease a new PR with a proper PR-branch
